tests/framework/utils_cpu_templates.py

```diff
@@ -19,6 +19,10 @@ def get_supported_cpu_templates():
     """
     vendor = cpuid_utils.get_cpu_vendor()
     if vendor == cpuid_utils.CpuVendor.INTEL:
+        # T2CL template is only supported on Cascade Lake and newer CPUs.
+        skylake_model = "Intel(R) Xeon(R) Platinum 8175M CPU @ 2.50GHz"
```

Comment:
can we make this a list named t2cl_exclusion_list in there are other models incompatible with t2cl?

Reply:
I think we can convert it if such use case arises. So far it looks simpler this way. Do you have a strong argument for making it a list now?